src/components/FiscalYear/FiscalYearDetails/FiscalYearDetailsContainer.js

```diff
@@ -72,8 +88,10 @@ const FiscalYearDetailsContainer = ({
   const removeFiscalYear = useCallback(
     () => {
       mutator.fiscalYear.DELETE(fiscalYear)
-        .then(() => history.goBack());
+        .then(showToast('ui-finance.fiscalYear.actions.remove.success'))
+        .then(() => history.push(FISCAL_YEAR_ROUTE));
```

Comment:
what is the reason of the second `.then()` ? it's not a promise chaining

Reply:
Fixed

Reply:
👍 fixed